src/structures/GuildAuditLogs.js

```diff
@@ -84,7 +84,7 @@ const Targets = {
 /**
  * All available actions keyed under their names to their numeric values.
  * @name GuildAuditLogs.Actions
- * @type {AuditLogAction}
+ * @type {Object<string, number>}
```

Comment:
Why were these changed? IIRC these typedefs exist, right?

Reply:
that was changed because that doc isn't the actual typedef for `AuditLogAction` and it was causing `GuildAuditLogs.Actions` to be typed as `string|number` on the docs, which is incorrect, its an object with keys as strings and numbers for values